mars/tensor/base/unique.py

```diff
@@ -242,6 +242,8 @@ def _tile_via_shuffle(cls, op):
             chunks = reduce_op.new_chunks(
                 [shuffle_chunk], kws=kws, order=op.outputs[0].order
             )
+            if op.return_inverse:
+                chunks[0].is_mapper, chunks[1].is_mapper = False, True
```

Comment:
Why only setting the `is_mapper` for this operand? Is it possible missing set the `is_mapper` attribute for some operands which mapper output are mixing with non shuffle data?

Reply:
`is_mapper` should be done in every operand which has shuffle and normal outputs mixed. It can't be set in a general way since we can't know which operands has such output pattern